target/linux/ath79/image/generic.mk

```diff
@@ -1763,7 +1763,7 @@ define Device/netgear_wndap360
   $(Device/netgear_generic)
   SOC := ar7161
   DEVICE_MODEL := WNDAP360
-  DEVICE_PACKAGES := kmod-leds-reset kmod-owl-loader
```

Comment:
not handy of ath9k. Why?

Reply:
Why should I need the owl-loader when the calibration data is directly extract from rom by the driver? Further, I have nor-rom. Not sure what you mean?